lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -108,8 +108,9 @@ dnsclient:
 domain_name:
   domain_name:
     # NOTE: This does not account for vrf ip domain-name
-    config_get: "show running-config"
+    config_get: "show running-config | include 'ip domain-name' | exclude 'use-vrf'"
```

Comment:
The 'ip domain-name' command doesn't take a 'use-vrf' parameter. The configuration you want to look out for is:

```
ip domain-name cisco.com
vrf context red
  ip domain-name red.com
vrf context green
  ip domain-name green.com
```